draft-ietf-ice-rfc5245bis.xml

```diff
@@ -3171,56 +3171,28 @@ Procedures for sending media differ for full and lite implementations.
 <section title="Procedures for Full Implementations">
 
 <t>
-Agents always send media using a candidate pair, called the selected
-candidate pair. An agent will send media to the remote candidate in
-the selected pair (setting the destination address and port of the
-packet equal to that remote candidate), and will send it from the
-local candidate of the selected pair. When the local candidate is
+A full implementation MUST NOT send media until it has a Valid list
+that contains a candidate pair for each component of that media
+stream. Once that happens, the agent MAY begin sending media
+packets.
```

Comment:
May want to clarify that the agent may only begin sending media packets "using candidate pairs in the Valid list".